src/main/java/org/myrobotlab/process/InProcessCli.java

```diff
@@ -386,15 +386,15 @@ public synchronized void stop() {
       worker.interrupt();
     }
 
-    if (in != null) {
+    if (in != null && in != System.in) {
```

Comment:
I think System.in.equals(in) would be better?

Reply:
k